.gitignore

```diff
@@ -40,6 +40,7 @@ Example/Pods/*
 !Example/Pods/Pods.xcodeproj/project.pbxproj
 Example/compile_commands.json
 Example/Emission/Configuration.h
+Artsy-UIFonts/*
```

Comment:
@ashkan18 - Saw that you updated the troubleshooting docs with this fix, but didn't add to `.gitignore` -- deliberate?

Reply:
Wait, what docs and why would there be this dir in the repo root?

Reply:
@alloy - oh sorry! i should have linked. I'm referring to the [troubleshooting](https://github.com/artsy/emission/blob/master/docs/troubleshooting.md) document. I ran into this issue last night.

Reply:
sorry for late response, yes! I ran into this originally when setting up Emission for the first time, @ashfurrow suggested to fetch `ArtsyFonts` from git so I documented it, it makes sense to add it to `.gitignore` if we really need to do it. It would be even better if there was a way to skip this folder all together though.

Reply:
Hmm, I don’t understand this fix. @ashfurrow can you explain when you have some time?